_plugins/related_posts.rb

```diff
@@ -4,6 +4,7 @@ class RelGenerator < Liquid::Tag
     def initialize(related_posts, text, tokens)
       super
       @params = text.split(' ')
+      @baseurl = Jekyll.sites[0].config['baseurl']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.